### ISSUE TITLE
Deposits: It is not clear to merchants that pending balances will not be paid out

### ISSUE THREAD

**Issue by brucealdridge** [Automattic/woocommerce-payments#7879]
### Describe the bug

When a merchant has a $0 or negative available balance it is not clear that the pending balance will not be paid out.

We should add a notice on the deposits overview page with clear wording. Something like 

```
Deposits are paused until your available balance is above $0.
```

related to #7876

**Comment by csmcneill** [comment]
`Deposits are paused` makes it sound like there was a risk action taken against the account. We'll want to adjust the wording a little to avoid sending mixed messages.

**Comment by brucealdridge** [comment]
> We'll want to adjust the wording a little to avoid sending mixed messages.

Current wording is: `Deposits may be interrupted while your %s balance remains negative. Why?`

looks like this might just work with changes in #7875

**Comment by brucealdridge** [comment]
.. actually, that is a negative notice, will need something for $0

**Comment by Jinksi** [comment]
Closing as resolved in 6.9.2 #7894
